conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+
+
+class cmockalibConan(ConanFile):
+    name = "cmocka"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Elearcommonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = "shared=False"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        if (self.settings.os == "Android"):
+            cmake.definitions["Platform"] = "android"
+        cmake.configure(source_folder=".")
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="include")
+        self.copy("*", dst="lib", src="lib", keep_path=False)
```

Comment:
Does this also copy the *.h files into lib? Any way to avoid this, by selectively copying *.so and *.a ?

Reply:
> Does this also copy the *.h files into lib? Any way to avoid this, by selectively copying *.so and *.a ?
No, it copies `*.h files into include in package` and `*.so and *.a into lib`